python/MaterialX/__init__.py

```diff
@@ -1,3 +1,24 @@
+# Python 3.8+ on Windows: DLL search paths for dependent
+# shared libraries
+# Refs.:
+# - https://github.com/python/cpython/issues/80266
+# - https://docs.python.org/3.8/library/os.html#os.add_dll_directory
+import os
+import sys
+if sys.platform == "win32" and sys.version_info >= (3, 8):
```

Comment:
Should we handle Python less than 3.8 by appending to `PATH` or `sys.path`?

Reply:
Probably not. It was never done before and this code only became necessary in 3.8 when accessing PATH for library lookups was disabled because it was quite a large security risk.